cache/sharded_cache.h

```diff
@@ -12,26 +12,30 @@
 #include <atomic>
 #include <string>
 
+#include "cache_key.h"
 #include "port/port.h"
 #include "rocksdb/cache.h"
 
 namespace ROCKSDB_NAMESPACE {
 
+typedef std::array<uint32_t, kCacheKeySize / 4> hash_t;
+
 // Single cache shard interface.
 class CacheShard {
```

Comment:
These changes potentially affect performance of LRUCache, due to another virtual call and moving a little more data around. We need to validate whether there's a measurable impact.

Reply:
I will run our cache_bench benchmarks to verify there's no regression.